src/Set/Extra.elm

```diff
@@ -1,25 +1,18 @@
-module Set.Extra
-    exposing
-        ( concatMap
-        , filterMap
-        , subset
-        , toggle
-        )
+module Set.Extra exposing (concatMap, filterMap, subset, toggle)
 
 {-| Convenience functions for working with Set.
 
 @docs concatMap, filterMap, subset, toggle
 
-    -- This module is used in the examples
```

Comment:
I was unable to add `import Set exposing (Set)` globally. It won't generate right tests from that :/ 

Any trick here?

Reply:
Nope, we remove support for global imports. Because it's nice to see all relevant imports next to the examples.